src/test/ts/module/Utils.ts

```diff
@@ -0,0 +1,35 @@
+import { Mouse, UiControls, UiFinder } from "@ephox/agar";
+import { TinyUiActions } from "@ephox/mcagar";
+import { Editor } from "@ephox/mcagar/lib/main/ts/ephox/mcagar/alien/EditorTypes";
+import { SugarBody, SugarElement, Traverse } from "@ephox/sugar";
+
+const pSetListBoxItem = async (editor: Editor, group: string, itemText: string): Promise<void> => {
+  const dialog = await TinyUiActions.pWaitForDialog(editor);
+  const element = UiFinder.findIn(dialog, 'label.tox-label:contains(' + group + ') + div.tox-listboxfield > .tox-listbox').getOrDie();
+  Mouse.click(element);
+  const list = await UiFinder.pWaitForVisible('Wait for list to open', SugarBody.body(), '.tox-menu.tox-collection--list');
+  const item = UiFinder.findIn(list, '.tox-collection__item-label:contains(' + itemText + ')').getOrDie();
+  const parent = Traverse.parent(item).getOrDie('Failed to find parent');
+  Mouse.click(parent);
+};
+
+const pSetInputFieldValue = async (editor: Editor, group: string, newValue: string): Promise<void> => {
+  const dialog = await TinyUiActions.pWaitForDialog(editor);
+  const element = UiFinder.findIn<HTMLInputElement>(dialog, 'label:contains("' + group + '") + input').getOrDie();
```

Comment:
We seem to use string interpolation (https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html) pretty heavily in tinymce's codebase. It's a good habit to get into. I'm not really into it yet, but I'm getting better.